src/components/Main/Main.js

```diff
@@ -8,9 +10,14 @@ import styles from './Main.module.scss';
  * @param {React.ReactElement} props.children The children to be rendered.
  * @returns {React.ReactElement} The Main component.
  */
-export default function Main({ children, className }) {
+export default function Main({ children, className, ...props }) {
   return (
-    <main className={[styles.main, className || undefined].join(' ')}>
+    <main
+      id={SELECTOR.MAIN_CONTENT_ID}
+      tabIndex="-1"
+      className={[styles.main, className || undefined].join(' ')}
```

Comment:
This does not need  the `undefined`

```suggestion
      className={[styles.main, className].join(' ')}
```

Reply:
Thanks @theodesp will remove